docs/docs/get-started.mdx

```diff
@@ -1,44 +1,106 @@
 ---
 id: get-started
-title: Become an Ory Insider
-sidebar_label: Ory Insider Program
+title: Ory Early Access
+sidebar_label: Ory Early Access
 ---
 
-Ory Cloud is currently in [early access](./early-access.mdx) and invite-only. To
-get access to Ory Cloud's full functionality you need to apply for the Ory
-Insider program. Ory Insiders
+Ory Cloud is currently in [early access](./early-access.mdx) and invite-only.  
+We are giving away one year for free for startups that want to migrate to Ory
+Cloud.  
+Limited seats, apply today: [console.ory.sh](https://console.ory.sh/).
 
-- have exclusive and unmetered access to Ory Cloud.
-- are invited to weekly meetings with Ory's core development team.
-- can influence the roadmap and priorities.
-- get exclusive Ory swag.
+### What you get
 
-The Ory Insider program is
-[priced at 99$ per month](https://www.ory.sh/pricing/). This price does not
-reflect the eventual price of using and running Ory as we aim to provide the
-most affordable platform out there!
+- 1 year free of charge Ory Cloud Project on a Developer plan.
 
-To apply for Ory Insiders, head over to [console.ory.sh](https://console.ory.sh)
-and sign up for an account.
+Features:
 
-![Sign up for Ory](./images/get-started/signup.png)
+- Login
+- Registration
+- APIs for Web App
+- APIs for Single Page App
+- APIs for Mobile App
+- Secure Account Recovery
+- E-Mail Verification
+- Breached Password Detection
+- Bring your own UI
+- User Management
+- Custom Identity Models
+- Standard 2FA
+- FIDO2-compliant 2FA
+- Instant Log Out
+- Encryption in Transit
+- Administrate Identities, Cookies & Sessions
+- Pay for what you use in production (fair use)
 
-## Become an Ory Insider
+and the following support options:
+
+- Dedicated Developer Relations
+- Chat support via Slack channel
+- Ticket via GitHub Issues
+
+A ton of other features, including Ory Hydra and Ory Keto functionality, are
+[on the roadmap](./early-access.mdx) and will be worked on in the course of the
+Early Access.
+
+### Where is the catch
```

Comment:
Maybe "What to expect" instead? Where's the catch is kinda negative

Reply:
```suggestion
### What to expect
```